src/Features/CSharp/Portable/AliasAmbiguousType/CSharpAliasAmbiguousTypeCodeFixProvider.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.AliasAmbiguousType;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp.AliasAmbiguousType
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.AliasAmbiguousType), Shared]
+    [ExtensionOrder(After = PredefinedCodeFixProviderNames.FullyQualify)]
+    internal class CSharpAliasAmbiguousTypeCodeFixProvider : AbstractAliasAmbiguousTypeCodeFixProvider
+    {
+        /// <summary>
+        /// 'reference' is an ambiguous reference between 'identifier' and 'identifier'
+        /// </summary>
+        private const string CS0104 = nameof(CS0104);
+
+        public override ImmutableArray<string> FixableDiagnosticIds
+            => ImmutableArray.Create(CS0104);
+
+        protected override string GetTextPreviewOfChange(SyntaxNode aliasNode)
+        {
+            Debug.Assert(aliasNode is UsingDirectiveSyntax);
+            // A poor man's name simplifier. For the preview of the context menu text the likely change is predicted by 
+            // removing the global:: namespace alias if present. For the majority of cases this should be the same result
+            // as what the real Simplifier produces in the preview pane and when the fix is applied.
+            aliasNode = RemoveGlobalNamespaceAliasIfPresent(aliasNode);
+            return aliasNode.NormalizeWhitespace().ToFullString();
+        }
+
+        NameSyntax GetLeftmostQualifiedName(NameSyntax nameSyntax)
```

Comment:
An easier approach would just be to do something like (from memory, so may have naming issues): 

```c#
aliasNode.ReplaceNodes(
    aliasNode.DescendentNodes().OfType<AliasQualifiedNode>().Where(a => a.Alias.IsGlobal),
    (n1, n2) => ((AliasQualifiedName)n1). Name);
```

Basically, you can just use 'ReplaceNodes' and say "replace all the alias nodes with their right side".